ledger/simulation/simulation_eval_test.go

```diff
@@ -47,6 +48,7 @@ func uint64ToBytes(num uint64) []byte {
 
 type simulationTestCase struct {
 	input         simulation.Request
+	nodeConfig    *config.Local
```

Comment:
If we want to move away from passing `config.Local` to `MakeSimulator`, this should also be changed here.

I'd be fine if this were just `DeveloperAPI bool`, and defaulting to false if not provided seems fine to me. We could then remove the code in `simulationTest` that calls to `config.GetDefaultLocal()`.